tests/command_db_options/tests.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase
+
+
+class CommandDBOptionChoiceTests(TestCase):
```

Comment:
Can you move this into `tests/admin_scripts/tests.py` and update from `TestCase` to `SimpleTestCase`?

Reply:
Thank you for your review. :) I will revise and commit to what you requested! :)